ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java

```diff
@@ -6,53 +6,44 @@
  */
 package com.connexta.ingest.transform;
 
+import com.connexta.transformation.rest.models.TransformRequest;
+import com.connexta.transformation.rest.models.TransformResponse;
 import java.net.URI;
 import java.net.URISyntaxException;
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
-import org.springframework.web.client.ResponseErrorHandler;
 import org.springframework.web.client.RestTemplate;
 
 @Service
 public class TransformClient {
 
-  private final RestTemplate restTemplate;
-  private final URI transformEndpoint = new URI("TODO");
+  private static final Logger LOGGER = LoggerFactory.getLogger(TransformClient.class);
 
-  @Autowired
-  public TransformClient(RestTemplate restTemplate) throws URISyntaxException {
-    this.restTemplate = restTemplate;
-  }
+  @Autowired private RestTemplate restTemplate;
 
-  public TransformResponse requestTransform(TransformRequest transformRequest) {
-    ResponseErrorHandler originalRequestHandler = restTemplate.getErrorHandler();
-    try {
-      restTemplate.setErrorHandler(new NoOpResponseErrorHandler());
-      return postForTransform(transformRequest);
-    } finally {
-      restTemplate.setErrorHandler(originalRequestHandler);
-    }
+  private URI transformEndpoint;
```

Comment:
❓ I'm a little confused on how this value gets injected. 
I don't see any autowiring or anything setting this through dependency injection

Reply:
We are not as far as injecting it. We are manually setting it for now.